test/tests/functional/pbs_hook_execjob_prologue.py

```diff
@@ -247,12 +250,20 @@ def test_prologue_hook_does_not_execute_twice_on_pbsdsh(self):
         attr = {'event': 'execjob_prologue'}
         self.server.create_import_hook(hook_name, attr, hook_body)
 
-        j = Job(TEST_USER, {'Resource_List.select': '2:ncpus=1',
-                            'Resource_List.place': 'scatter'})
-        j.create_script('pbsdsh -n 1 hostname')
+        j = Job(PBSROOT_USER, {'Resource_List.select': '2:ncpus=1',
+                               'Resource_List.place': 'scatter'})
+        j.create_script('pbsdsh  hostname')
         jid = self.server.submit(j)
+        attribs = self.server.status(JOB, id=jid)
         self.server.expect(JOB, 'queue', op=UNSET, id=jid)
-        match = self.momB.log_match("Job;%s;executed prologue hook" % jid,
-                                    allmatch=True)
-        self.assertTrue(match is not None)
-        self.assertTrue(len(match) == 1)
+        host, opath = attribs[0]['Output_Path'].split(':', 1)
```

Comment:
if the job status is unset how will you get the value of output_path? you need to read it from the server_logs or enable job history or add a sleep for 10-12s in the job script.

Reply:
@anamikau I believe the output file will be available only when the job is finished right? And by this, I'm actually getting the contents of the output file after the job has finished.
Let me know if I'm missing something.

Reply:
hey output_path will be set as soon as the job starts. you are reading this from status/qstat. once the job is done you won't get any output from qstat. you are getting this just by luck I would say as the systems you will be using are fast. But it might not be true all the time.